docs/en/ingest-arch/15-proxy.asciidoc

```diff
@@ -1,14 +1,19 @@
 [[agent-proxy]]
 == {agent} to proxy to {es}
 
-image::images/ea-proxy-es.png[Image showing connections between {agent} and {es} using a proxy]
+image::images/ea-proxy-fs-es.png[Image showing connections between {agent} and {es} using a proxy when Fleet Server is managed by Elastic]
+
+image::images/ea-fs-proxy-es.png[Image showing connections between {agent} and {es} using a proxy]
```

Comment:
Note to self @karenzone: Adding both architectures to same topic until we determine if there are enough differences to warrant separate child  topics